docs/topics/mta-web-config-jira-credentials.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-credentials_{context}"]
+= Configuring Jira credentials
+
+To define a Jira instance in {ProductShortName} and establish a connection to that instance, you must create an {ProductShortName} credential to authenticate to the Jira instance's API.
+
+Two types of credentials are available:
+
+* *Basic auth* - for Jira Cloud and a private Jira Server or Datacenter
```

Comment:
Lookedlike Jira Software Server and Jira Software Data Center (two words) when I Googled the phrases.